### PR TITLE
Increase minimum pop for blob

### PR DESCRIPTION
**Pull request by JamieD1**
On low pop rounds its a landslide and unfair to the crew.

:cl:  JamieD12
tweak: Changed blob from 20 to 25 players
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
Can you change the Pr name to reflect what this actually does

File: code/modules/events/blob.dm [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	weight = 10
 	max_occurrences = 1
 
-	min_players = 20
+	min_players = 30
```
Comment (Readystorm546) [inline code/modules/events/blob.dm]: I would  not do a drastic change to 30. How about 25 instead

Comment (JamieD1) [reply code/modules/events/blob.dm]: We could try 25 and see how it goes

Comment (boodaliboo) [reply code/modules/events/blob.dm]: please reflect that you changed it to 25 in the descrption

Comment (boodaliboo) [reply code/modules/events/blob.dm]: please reflect that you changed it to 25 in the descrption

**Pull request merged**